generation/readme_update.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# This script should run as a part of new-library generation process.
```

Comment:
Can you add corresponding change in generation/new_client in this pull request?

Reply:
Done.